tests/puppeteer/campaigns/sanity/01_installShop/01_installShop.js

```diff
@@ -27,39 +29,45 @@ describe('Install Prestashop', async () => {
   // Steps
   it('should open the Install page', async function () {
     await this.pageObjects.installPage.goTo(global.INSTALL.URL);
-    await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
+    const result = await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
       [this.pageObjects.installPage.firstStepFrTitle, this.pageObjects.installPage.firstStepEnTitle]);
+    await expect(result).to.be.true;
   });
   it('should change language to English and check title', async function () {
     await this.pageObjects.installPage.setInstallLanguage();
-    await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
+    const result = await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.firstStepPageTitle,
       this.pageObjects.installPage.firstStepEnTitle);
+    await expect(result).to.be.true;
   });
   it('should click on next and go to step \'License Agreements\'', async function () {
     await this.pageObjects.installPage.nextStep();
-    await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.secondStepPageTitle,
+    const result = await this.pageObjects.installPage.checkStepTitle(this.pageObjects.installPage.secondStepPageTitle,
```

Comment:
Same here, don't hesitate to pass one parameter per line when it's very long like this

Reply:
done thanks @matthieu-rolland